docs/news.rst

```diff
@@ -3,6 +3,336 @@
 Release notes
 =============
 
+1.1.0 (unreleased)
+------------------
+
+Python 3 Support (basic)
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+We have been hard at work to make Scrapy work on Python 3. Some features
+are still missing (and may never be ported to Python 3, see below),
+but you can now run spiders on Python 3.3, 3.4 and 3.5.
+
+Almost all of addons/middleware should work, but here are the current
+limitations we know of:
+
+- s3 downloads are not supported (see :issue:`1718`)
+- sending emails is not supported
+- FTP download handler is not supported (non-Python 3 ported Twisted dependency)
+- telnet is not supported (non-Python 3 ported Twisted dependency)
+- there are problems with non-ASCII URLs in Python 3
+- reported problems with HTTP cache created in Python 2.x which can't be used in 3.x (to be checked)
+- there is also a nasty issue with cryptography library:
+  recent versions don't work well on OS X  + Python 3.5 (see https://github.com/pyca/cryptography/issues/2690),
+  downgrading to an older version helps
+
+
+New Features and Enhancements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Command line tool has completion for zsh (:issue:`934`)
+- ``scrapy shell`` works with local files again ; this was a regression
+  identified in 1.0+ releases (:issue:`1710`, :issue:`1550`)
+- ``scrapy shell`` now also checks a  new ``SCRAPY_PYTHON_SHELL`` environment
+  variable to launch the interactive shell of your choice ;
+  ``bpython`` is a newly supported option too (:issue:`1444`)
+- Autothrottle has gotten a code cleanup and better docs ;
+  there's also a new ``AUTOTHROTTLE_TARGET_CONCURRENCY`` setting which
+  allows to send more than 1 concurrent request on average (:issue:`1324`)
+- Memory usage extension has a new ``MEMUSAGE_CHECK_INTERVAL_SECONDS``
+  setting to change default check interval (:issue:`1282`)
+- HTTP caching follows RFC2616 more closely (TODO: link to docs);
+  2 new settings can be used to control level of compliancy:
+  ``HTTPCACHE_ALWAYS_STORE`` and ``HTTPCACHE_IGNORE_RESPONSE_CACHE_CONTROLS``
+  (:issue:`1151`)
+- Scheme Download handlers are now lazy-loaded on first request using
+  that scheme (``http(s)://``, ``ftp://``, ``file://``, ``s3://``)
+  (:issue:`1390`, :issue:`1421`)
+- RedirectMiddleware now skips status codes in ``handle_httpstatus_list``,
+  set either as spider  attribute or ``Request``'s ``meta`` key
+  (:issue:`1334`, :issue:`1364`, :issue:`1447`)
+
+
+- Form submit button plain #1469 (https://github.com/scrapy/scrapy/commit/b876755f1cee619d8c421357777d223037d5289c)
+    Fixes: Form submit button (https://github.com/scrapy/scrapy/issues/1354)
+- Implement FormRequest.from_response CSS support #1382 (https://github.com/scrapy/scrapy/commit/a6e5c848feb672c117f3380976077b6d0f42e3a6)
+ + Fix version number to appear new feature #1706
```

Comment:
I think minor fixes like this don't need to appear in the release notes (this fix a doc issue that wasn't in 1.0.4 in first place).